storage/plugins/ncip-server/wrapper.php

```
@@ -0,0 +1,58 @@
+            \App\Support\SecureLogger::debug('[NcipServer] Plugin activated');
+        }
+
+        public function onDeactivate(): void
+        {
+            $this->instance->onDeactivate();
+            \App\Support\SecureLogger::debug('[NcipServer] Plugin deactivated');
+        }
+
+        public function onInstall(): void
+        {
+            \App\Support\SecureLogger::debug('[NcipServer] Plugin installed');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'onInstall'. (undefined)

(StaticAccess)